### ISSUE TITLE
Test failure test_metadata

### ISSUE THREAD

**Issue by sciunto** [soft-matter/pims#202]
Hi,

I'm packaging for archlinux v0.3.2. I got this (py2 and 3)

```
FAIL: test_metadata (pims.tests.test_common.TestTiffStack_pil)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/fr/aur4/python2-pims/src/pims-0.3.2/pims/tests/test_common.py", line 536, in test_metadata
    assert_equal(m, d)
  File "/usr/lib/python2.7/site-packages/numpy/testing/utils.py", line 281, in assert_equal
    assert_equal(len(actual), len(desired), err_msg, verbose)
  File "/usr/lib/python2.7/site-packages/numpy/testing/utils.py", line 354, in assert_equal
    raise AssertionError(msg)
AssertionError: 
Items are not equal:
 ACTUAL: 2
 DESIRED: 3
```

**Comment by danielballan** [comment]
This is a known issue with pillow >=3. New conda packages for pims will restrict pillow < 3. We can't restrict it in the setup.py because pillow is an optional dep.

Once that's done we'll go about resolving the underlying issue upstream. Do you have any time to look into it?

**Comment by caspervdw** [comment]
Solved in #204
